### PR TITLE
bugfix

### PR DESCRIPTION
**Pull request by agentmario**

### CODE REVIEW

File: views/app.cjsx [hunk 1]
```diff
@@ -276,10 +276,10 @@ PoiControl = React.createClass
         <Button onClick={@handleOpenDevTools} onContextMenu={@handleOpenWebviewDevTools} bsSize='small'><FontAwesome name='gears' /></Button>
       </OverlayTrigger>
       <OverlayTrigger placement='left' overlay={<Tooltip>缓存目录</Tooltip>}>
-        <Button onClick={@handleOpenCacheFolder} bsSize='small'><FontAwesome name='bolt' /></Button>
+        <Button onClick={@handleOpenCacheFolder}  onContextMenu={@handleOpenMakaiFolder bsSize='small'><FontAwesome name='bolt' /></Button>
```
Comment (alvin-777) [inline views/app.cjsx]: - [ ] Missing '}'

File: views/app.cjsx [hunk 2]
```diff
@@ -276,10 +276,10 @@ PoiControl = React.createClass
...
       </OverlayTrigger>
       <OverlayTrigger placement='left' overlay={<Tooltip>截图目录</Tooltip>}>
-        <Button onClick={@handleOpenScreenshotFolder} onContextMenu={@handleOpenMakaiFolder} bsSize='small'><FontAwesome name='photo' /></Button>
+        <Button onClick={@handleOpenScreenshotFolder}} bsSize='small'><FontAwesome name='photo' /></Button>
```
Comment (alvin-777) [inline views/app.cjsx]: - [ ] Too many '}'

**Comment by alvin-777** [comment]
Well OK I see you've created another pull request...
As someone has already mentioned, you should keep working on this branch, and the changes will be automatically added to this pull request. Otherwise it'll just confuse people who's reviewing your code.

**Comment by yukixz** [comment]
Merged PR won't be updated.
For merged PR, creating a new PR is right.

**Comment by alvin-777** [comment]
Alright, fair enough.

**Pull request merged**